### PR TITLE
Allow custom styling to ScrollableTabBar

### PR DESCRIPTION
**Pull request by nimzco**
I really wanted to lower the padding of the tabs so I thought, why not allow custom styling to the tab bar?

### CODE REVIEW

File: ScrollableTabBar.js [hunk 1]
```diff
@@ -33,9 +33,11 @@ const ScrollableTabBar = React.createClass({
     activeTextColor: React.PropTypes.string,
     inactiveTextColor: React.PropTypes.string,
     scrollOffset: React.PropTypes.number,
+    style: React.PropTypes.object,
   },
 
   getInitialState() {
+    this.props.style = this.props.style || {}
```
Comment (skv-headless) [inline ScrollableTabBar.js]: please move to getDefaultProps

Comment (nimzco) [inline ScrollableTabBar.js]: Good catch my bad.

**Comment by skv-headless** [comment]
In my project we use same api for passing styles inside component. But for library I would prefer
```this.props.containerStyle``` instead of ```this.props.style.container```. I hope it would not be so difficult fix that.

**Comment by nimzco** [comment]
That's good for me too, I'll change that right away.

**Comment by nimzco** [comment]
Like that: https://github.com/skv-headless/react-native-scrollable-tab-view/pull/257/commits/7ab5444ea88ebbd7fad8c38d01c3b2e71a0cce7c ?

**Comment by skv-headless** [comment]
Yep. Sorry one more thing. Use ```style``` instead of ```containerStyle``` to make it similar to https://github.com/skv-headless/react-native-scrollable-tab-view/blob/master/DefaultTabBar.js#L59

**Comment by nimzco** [comment]
Like that then: https://github.com/skv-headless/react-native-scrollable-tab-view/pull/257/commits/ce89436ed2d53756ba189b6b23f6a9196e74ce77 ?

**Comment by skv-headless** [comment]
👍

File: ScrollableTabBar.js [hunk 2]
```diff
@@ -161,7 +167,7 @@ const ScrollableTabBar = React.createClass({
         bounces={false}
       >
         <View
-          style={[styles.tabs, {width: this.state._containerWidth, }, ]}
+          style={[styles.tabs, {width: this.state._containerWidth, }, this.props.tabsStyle]}
```
Comment (skv-headless) [inline ScrollableTabBar.js]: maybe ```tabsContainer``` would be better name?

Comment (nimzco) [inline ScrollableTabBar.js]: Agreed. `sS` could generate a lot of pain in the future...

Comment (skv-headless) [inline ScrollableTabBar.js]: I just think that it is not very intuitive.

**Comment by skv-headless** [comment]
Thanks

**Pull request merged**